packages/steveo/src/consumers/kafka.ts

```diff
@@ -240,11 +243,10 @@ class KafkaRunner
   };
 
   consumeCallback = async (err, messages) => {
-    this.logger.debug(`Consumer callback: ${messages?.[0]}`);
+    this.logger.debug('Consumer callback: ', messages?.[0]);
```

Comment:
Personally I think this should be at a `verbose` level, but just correcting the argument order

```suggestion
    this.logger.debug({ messages }, 'Consumer callback: ');
```